spring-kafka/src/main/java/org/springframework/kafka/support/Acknowledgment.java

```diff
@@ -43,11 +43,11 @@ public interface Acknowledgment {
 	 * {@code sleep + time spent processing the previous messages from the poll} must be
 	 * less than the consumer {@code max.poll.interval.ms} property, to avoid a
 	 * rebalance.</b>
```

Comment:
Btw, is it still true after the `sleep()` was removed in https://github.com/spring-projects/spring-kafka/pull/2131?

Reply:
It is no longer true; those sentences can be removed.

Reply:
I removed the sentence and precised millis also in the generic JavaDoc descirption.